src/style.css

```diff
@@ -1 +1,69 @@
 /* Styles go here */
+
+body {
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  position: absolute;
+  top: 0;
+  left: 0;
+
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  width: 100%;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  padding-left: 50px;
+}
+
+.nav__list {
+  margin: 0;
+  display: flex;
+  align-items: center;
+
+  padding: 0 40px;
+  list-style: none;
+}
+
+.nav__item {
+  height: 60px;
+  display: flex;
+}
```

Comment:
Why do we need flex here?

I think we should set the height to the link to make it fully clickable but not to the item